.devcontainer/.install_utils/install_acados.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Define the acados installation path
+ACADOS_ROOT=${1:-/acados}
+
+# Clone acados into the target folder
+git clone https://github.com/acados/acados.git $ACADOS_ROOT
+cd $ACADOS_ROOT
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)